lib/seko_ecom_api/clients/omni_returns_client.rb

```diff
@@ -48,10 +50,13 @@ def handle_response(response)
       parsed_response
     end
 
+    # need to be manual parsed here instead of in the middleware
+    # since Seko response's Content-Type is always in 'text/html'
+    # with the response body can be in JSON or in html
     def parse_response(response)
       parsed_response = JSON.parse(response.body)
       parsed_response.deep_transform_keys(&:underscore)
-    rescue JSON::ParserError
+    rescue JSON::ParserError, NoMethodError
```

Comment:
Why do we have to rescue NoMethodError here?

Reply:
Oh I forgot to remove it

Reply:
@trang-mai-nguyen But you haven't remove it lol

Reply:
@andychongyz omg what the heck lol. Removed it